docs/src/examples.md

```diff
@@ -0,0 +1,139 @@
+Examples
+========
+
+The K8sClusterManager is intended to be used inside a pod running on a Kubernetes cluster.
+
+## Launching an interactive session
+
+The following manifest will create a [Kubernetes job](https://kubernetes.io/docs/concepts/workloads/controllers/job/)
+named "interactive-session". This job will spawn a pod (see `spec.template.spec`) which will
+run an interactive Julia session with the latest release of K8sClusterManagers.jl installed.
+Be sure to create the required [service account and associated permissions](../patterns/#required-permissions)
+before proceeding.
+
+````@eval
+using Markdown
+Markdown.parse("""
+```yaml
+$(read("interactive-session.yaml", String))
```

Comment:
Should these reside in `docs/src` or live somewhere else?

Reply:
seems fine to me to have them in docs/src for now unless there's a benefit to having them elsewhere